packages/animation-curve-block/src/components/BlankSlate.tsx

```diff
@@ -0,0 +1,98 @@
+    const [description, setDescription] = useState('');
+    const [isFlyoutOpen, setIsFlyoutOpen] = useState(false);
+    const defaultAnimationCurve: AnimationCurve = {
+        id: generateRandomId(),
+        title: '',
+        description: '',
+        animationFunction: {
+            type: AnimationCurveType.EaseInOut,
+            parameters: defaultAnimationCurveTypeValues[AnimationCurveType.EaseInOut],
+            duration: 1.0,
+        },
+    };
+
+    const [animationCurve, setAnimationCurve] = useState<AnimationCurve>(defaultAnimationCurve);
+    const addAnimationCurve = async (id: string) => {
+        setLocalItems([...(content ?? []), { ...animationCurve, id }]);
+        await setBlockSettings({
+            content: [...(content ?? []), { ...animationCurve, id: animationCurve.id }],
+        });
+        setIsFlyoutOpen(false);
+        setTitle('');
+        setDescription('');
+    };
+
+    useEffect(() => {
+        setAnimationCurve({ ...animationCurve, title });
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [title]);
+
+    useEffect(() => {
+        setAnimationCurve({ ...animationCurve, description });
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [description]);
+
+    return (
+        <div data-test-id="blank-slate">
+            <AnimationCurveFlyout
+                animationCurve={animationCurve}
+                onSave={addAnimationCurve}
+                onCancel={() => setIsFlyoutOpen(false)}
+                onOpenChange={(isOpen) => setIsFlyoutOpen(isOpen)}
+                onAnimationCurveChange={setAnimationCurve}
+                isFlyoutOpen={isFlyoutOpen}
+                lineColor={lineColor}
+                endpointColor={endpointColor}
+                gridColor={gridColor}
+            />
+            <button
+                data-test-id="blank-slate-button"
+                style={{
+                    height,
+                }}
+                className="tw-flex tw-justify-center tw-items-center tw-p-7 tw-w-full tw-bg-blank-state-shaded-inverse tw-border tw-border-blank-state-line tw-rounded tw-text-blank-state-shaded hover:tw-text-blank-state-hover hover:tw-bg-blank-state-hover-inverse hover:tw-border-blank-state-line-hover active:tw-text-blank-state-pressed active:tw-bg-blank-state-pressed-inverse active:tw-border-blank-state-line-hover"
+                onClick={() => setIsFlyoutOpen(true)}
+            >
+                <span className="tw-mr-3">
+                    <IconPlus24 />
+                </span>
+                Add animation curve
+            </button>
+
+            <CardText
+                appBridge={appBridge}
+                title={title}
+                description={description}
+                hasBorder={hasBorder}
+                designTokens={designTokens}
+                setDescription={setDescription}
+                setTitle={setTitle}
+                updateValueOnChange={true}
```

Comment:
Can we save some API requests and just save on blur all the time? That way you can also omit this prop and just set it to false in CardText internally.

Reply:
Yep I know that. I was just wondering what does it do because in Card.tsx it is set to false explicitly, but here it is set to true explicitly.

Reply:
Its needed for clearing the `BlankSlate -> CardText -> RTE's`, after adding a new animation curve, otherwise the `setTitle('')` or `setDescription('')` won't have any effects